### DISCUSSION TITLE
What should the FOLDER settings be to match Orig SLSK folders?

### DISCUSSION THREAD

**Discussion by DeathStalker77** [nicotine-plus/nicotine-plus#2695] [Q&A]
Orig SLSK has only 2 folders - DOWNLOADING & COMPLETE - no "INCOMPLETE".

Thanks!

--- DS

**Comment by slook** [comment]
Downloading = Incomplete
Complete = Finished

**Comment by DeathStalker77** [comment]
Might be good to make that clear - not sure why it was changed to begin with, that only complicates things.  KISS Principle :-)

**Reply by slook** [reply]
Afaik, it has not been changed, it has always been different from the official client. There probably isn't a good reason for that decision.

You can of course, set the folders whatever you wish on your filesystem.

**Comment by mathiascode** [comment]
I didn't choose the `incomplete` term (it's been there since the beginning), but it seems to be consistent with some BitTorrent clients.

Not sure if `downloading` is much better, feels like it would be easy to confuse with `downloaded` or `downloads` if you're not reading carefully.

DC++ uses `unfinished`.

**Comment by DeathStalker77** [comment]
I'm just thinking of a way to make the 2 compatible - for example if, like me, you started with SLSK then wanted to transition to N+, but still had stuff queued in SLSK and wanted to ensure you didn't DL duplicates.  Maybe a UI interface where you can match them up, so it's visually clearer?

**Reply by slook** [reply]
https://github.com/nicotine-plus/nicotine-plus/issues/1685
